redbot/cogs/streams/streamtypes.py

```diff
@@ -56,10 +56,11 @@ class Stream:
     token_name: ClassVar[Optional[str]] = None
 
     def __init__(self, **kwargs):
+        self._bot = kwargs.pop("_bot")
```

Comment:
This change breaks the `[p]twitchstream` and likely `[p]youtubestream` commands with a KeyError.